### PR TITLE
enabled external link checks;

### PR DESCRIPTION
**Pull request by mstbbs**
@irabinovitch - i have added a links.ignore file. you can add patterns that you want to be checked against found links. if the found link contains one of these patterns then it will be skipped.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ variables:
 # ================== copy scripts =============== #
 before_script:
   - source /usr/local/bin/helpers.sh  # source helpers so they are available in the container
+  - find local/etc/ -type f -exec cp {} /etc \;  # load configs
```
Comment (mstbbs) [inline .gitlab-ci.yml]: load the links.ignore file into the expected location in the container.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -114,7 +115,7 @@ test_live_images:
     URL: ${LIVE_DOMAIN}
   script:
     - pull_artifact_from_s3
-    - test_site_links "images" "${URL}" "True" "False" "True"
+    - test_site_links "images" "${URL}" "True" "True" "True"
```
Comment (mstbbs) [inline .gitlab-ci.yml]: These "True" args are: for check-all, check-external, verbose output. good to have on for the first little while

File: local/etc/links.ignore [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+tel:
+irc:
+file:
+localhost:
+url.to.snpashot.com
+app.datadoghq.com
+app.datadog.com
+some-account.atlassian.net
+bitbucket.org/groupname/reponame
+alert.victorops.com/integrations/datadog/20140523/alert
```
Comment (mstbbs) [inline local/etc/links.ignore]: the linkchecker will look skip any link that contains a pattern from this list

**Comment by irabinovitch** [comment]
Thanks Michael. LGTM.

**Pull request merged**